.gitignore

```diff
@@ -60,4 +60,5 @@ env[23]/
 
 # Project specific ignores
 static/js/modules/*.js
-static/js/dist/*.js
\ No newline at end of file
+static/js/dist/*.js
+/static/js/app.js
```

Comment:
I typically avoid the initial slash here, I prefer to use the relative path syntax

Reply:
👌🏻